pom.xml

```diff
@@ -18,6 +18,11 @@
 
     <properties>
         <connector.name>overrideme</connector.name>
+        <container.memory>512000000</container.memory>
+        <container.logging.level>DEBUG</container.logging.level>
```

Comment:
I think this goes away when you remove the --logging.level

Reply:
yes.. not required.. i have removed it.